pixl_core/tests/uploader/test_dicomweb.py

```diff
@@ -0,0 +1,73 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Test functionality to upload files to a DICOMWeb endpoint."""
+
+from __future__ import annotations
+
+from typing import Optional
+
+import pytest
+import requests
+from core.uploader._dicomweb import DicomWebUploader  # type: ignore [import-untyped]
+from decouple import config  # type: ignore [import-untyped]
+
+ORTHANC_URL = config("ORTHANC_URL")
+DICOM_ENDPOINT_NAME = config("DICOM_ENDPOINT_NAME")
+ORTHANC_USERNAME = config("ORTHANC_USERNAME")
+ORTHANC_PASSWORD = config("ORTHANC_PASSWORD")
+
+
+class MockDicomWebUploader(DicomWebUploader):
```

Comment:
tbf as this has some decent testing of dicom web here, could just not have the system test version as we know that the overall uploader flow works

Reply:
Hmm yeah, I actually thought of removing the unit test from core and keep the system test, as the latter is a bit more realistic, but I'm fine with either :)

Reply:
mm yeah, let's leave both in for now and we can always reassess how this affects development later on maybe?